PPM-383: remove unsupported 60ghz option from the core

DanielEdriPurpl opened the pull request:
Remove from the persistent DB all references to the unsupported 60GHz band.
This includes the BML CLI, the DB, and anything in between.

itayx commented on framework/tlvf/AutoGenerated/include/tlvf/WSC/eWscRfBands.h:
```diff
@@ -19,13 +19,8 @@
-}; // close namespace: WSC
+}; // namespace WSC
 
 #endif //_TLVF/WSC_EWSCRFBANDS_H_
```
The removal need to be only in the persistent DB

itayx commented on framework/tlvf/yaml/tlvf/WSC/eWscRfBands.yaml:
```diff
@@ -7,5 +7,3 @@ eWscRfBands:
   WSC_RF_BAND_2GHZ: 0x01
   WSC_RF_BAND_5GHZ: 0x02
-  WSC_RF_BAND_60GHZ: 0x04
-  
```
The removal need to be only in the persistent DB

itayx commented on framework/tlvf/yaml/tlvf/ieee_1905_1/tlvSupportedFreqBand.yaml:
```diff
@@ -16,4 +16,3 @@ eValue:
   _enum_storage: uint8_t
   BAND_2_4G: 0x00
   BAND_5G: 0x01
-  BAND_60G: 0x02
```
The removal need to be only in the persistent DB

itayx commented on controller/src/beerocks/master/son_master_thread.cpp:
```diff
@@ -430,10 +430,6 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_search(const std::string
-    case ieee1905_1::tlvAutoconfigFreqBand::IEEE_802_11_60_GHZ: {
-        tlvSupportedFreqBand->value() = ieee1905_1::tlvSupportedFreqBand::BAND_60G;
-        break;
-    }
```
The removal need to be only in the persistent DB

itayx commented on agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp:
```diff
@@ -2874,9 +2874,6 @@ bool backhaul_manager::handle_1905_autoconfiguration_response(ieee1905_1::CmduMe
             break;
-        case ieee1905_1::tlvSupportedFreqBand::BAND_60G:
-            LOG(DEBUG) << "received autoconfiguration response for 60GHz band, unsupported";
-            return true;
```
The removal need to be only in the persistent DB

itayx reviewed (changes requested):
Two things:
1) The removal of the 60GHz band needs to be contained to only the Persistent DB feature.
2) Please follow the guidelines of how commit messages are supposed to be structured as described [here](https://github.com/prplfoundation/prplMesh/blob/master/CONTRIBUTING.md#commits)

itayx reviewed (approved):
Looks good, please add PR description

mariomaz reviewed (approved):
LGTM

adam1985d commented on common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_common.h:
```diff
@@ -1342,23 +1342,21 @@ typedef struct sBssidInfo {
     uint32_t downlink_estimated_mac_data_rate_mbps;
     uint32_t uplink_estimated_mac_data_rate_mbps;
     uint8_t sta_measured_uplink_rssi_dbm_enc;
```
suggested commit message:
```
common: tlvf: remove the unsupported option of 60ghz

The 60Ghz band is not supported and should not be part of the possible
configuration of the selected bands.

Remove the 60Ghz from the eSelectedBands enum which is used by the
persistent DB to store the selected bands.

https://jira.prplfoundation.org/browse/PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>
```

adam1985d commented on controller/src/beerocks/bml/bml_defs.h:
```diff
@@ -126,7 +126,6 @@ extern "C" {
 #define BML_CLIENT_SELECTED_BANDS_24G 1
 #define BML_CLIENT_SELECTED_BANDS_5G 2
 #define BML_CLIENT_SELECTED_BANDS_6G 4
```
suggested commit message:
```
bml: remove the unsupported 60ghz option from selected bands

Remove the 60ghz option from the BML_CLIENT_SELECTED_BANDS in the
bml_defs and bml_cli.

https://jira.prplfoundation.org/browse/PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>
```

adam1985d reviewed (changes requested):
please fix commit messages and will re-review

DanielEdriPurpl committed:
common: tlvf: remove the unsupported option of 60Ghz

The 60Ghz band won't be supported for the time being,
and shouldn't be part of the available configuration
of the selected bands.

Remove the 60Ghz from the eSelectedBands enum which is
used by the presistent DB

PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>

DanielEdriPurpl committed:
bml: remove the unsupported 60Ghz option rom selected bands

Remove the 60Ghz option from the BML_CLIENT_SELECTED_BANDS in both
bml_defs and bml_cli respectively

PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>
